README.md

```diff
@@ -24,6 +24,12 @@ pub type Point3<T> = [T; 3];
 pub type Point4<T> = [T; 4];
 ~~~
 
+Gradient Noise
+--------------
+Gradient noise produces a smooth, continuous value over space. It's achieved by dividing space into regions, and placing a random gradient at each vertex, then blending between those gradients.
+
+**Perlin Noise**
```

Comment:
That produces much larger headers. It becomes much harder to distinguish them from the section headers they're under. I think it looks cleaner if they're just bold.